participants/iobert.json

```
@@ -0,0 +1,24 @@
+{
+	"realName": {
+		"givenName": "Marius",
+		"familyName": "Obert",
+		"placeFamilyNameFirst": false,
+		"hideFamilyNameOnWebsite": false
+	},
+	"githubAccountName": "iobert",
+	"company": "Twilio",
+	"when": {
+		"friday": true,
+		"saturday": false
+	},
+	"iCanTakeNotesDuringSessions": false,
+	"tags": ["JavaScript", "OpenAI", "NextJS", "TypeScript", "AI Dev Tools"],
+	"vegan": false,
+	"vegetarian": false,
+	"allergies": [],
+	"whatIsMyConnectionToJavascript": "It's my go-to language",
+	"whatCanIContribute": "I'm happy to host a session again",
+	"linkedin": "https://www.linkedin.com/in/mariusobert/",
```

I don't think it works because it should have a "null" as value. @Narigo this value is supposed to be optional right? could change the schema so that we can allows to omit the attribute

```suggestion
	"whatCanIContribute": "I'm happy to host a session again",
	"tShirtSize": null,
	"linkedin": "https://www.linkedin.com/in/mariusobert/",
```

---

Yes, we should change it on our side as well. But to make this work quickly, you can use `null` for value as a workaround now @IObert